cylc/rose/jinja2_parser.py

```diff
@@ -27,6 +29,155 @@
     Neg,
     Pos
 )
+import jinja2.lexer
+
+from cylc.flow import LOG
+
+
+def _strip_leading_zeros(string):
```

Comment:
Is it reasonable to have a function which wraps another function with almost no change?
Is it reasonable to have 4 tests which ought to be covered by lstrip's unit testing?

Reply:
* It's used twice so may as well centralise.
* The tests are more for documentation.

Reply:
Is there any risk of negative numbers being fed to this?

Reply:
I think the sign will be stripped by this point but give it a try.

Reply:
I'm happy with this having tried it.